treefmt.toml

```diff
@@ -0,0 +1,17 @@
+[formatter]
+
+[formatter.nix]
+command = "alejandra"
+excludes = ["yarn-project.nix"]
+includes = ["*.nix"]
+
+# TODO: would this conflict or just work (tm)?
+# [formatter.prettier]
+# command = "prettier"
+# includes = ["*.md", "*.toml", "*.yaml", "*.json"]
+# options = ["--plugin", "prettier-plugin-toml", "--write"]
+#
+# [formatter.shell]
+# command = "shfmt"
+# includes = ["*.sh"]
+# options = ["-i", "2", "-s", "-w"]
```

Comment:
Not ready to be merged

Reply:
@rhyslbw can you maybe help to answer the question of the TODO? Maybe point around on how this is currently wired.

Reply:
We currently have no linting covering those files, so it should just work, although when enabling we should also commit it's effect

Reply:
Attenuated (w/o `*.json`) & addressed in e4f6ed1e2e2c0b252d6ee62c70ae64c5276446a9